zetaclient/keys/keys.go

```diff
@@ -12,23 +12,25 @@ import (
 	cryptocodec "github.com/cosmos/cosmos-sdk/crypto/codec"
 	ckeys "github.com/cosmos/cosmos-sdk/crypto/keyring"
 	cryptotypes "github.com/cosmos/cosmos-sdk/crypto/types"
-	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkTypes "github.com/cosmos/cosmos-sdk/types"
 	"github.com/rs/zerolog/log"
+	"github.com/zeta-chain/zetacore/cmd"
 	"github.com/zeta-chain/zetacore/common"
 	"github.com/zeta-chain/zetacore/common/cosmos"
 	"github.com/zeta-chain/zetacore/zetaclient/config"
+	errors2 "github.com/zeta-chain/zetacore/zetaclient/errors"
```

Comment:
```suggestion
	zetaerrors "github.com/zeta-chain/zetacore/zetaclient/errors"
```
We should not use number in imports